ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -72,7 +79,19 @@
             disabled={{this.isSubmitDisabled}}
             data-test-sync-submit
           />
-          <Hds::Button @text="Back" @color="secondary" {{on "click" this.back}} data-test-sync-cancel />
+          {{#if this.syncedSecret}}
+            <Hds::Button
+              @text="View synced secrets"
+              @icon="chevron-right"
+              @iconPosition="trailing"
+              @color="tertiary"
+              @route="secrets.destinations.destination.secrets"
+              @model={{@destination}}
+              data-test-sync-cancel
+            />
+          {{else}}
+            <Hds::Button @text="Back" @color="secondary" {{on "click" this.back}} data-test-sync-cancel />
```

Comment:
The `back` method for the Back button transitions to the same route so perhaps we should remove that method and use the route and model args? I'm wondering if we really need that button or if just having the new one display all the time is fine?

Reply:
Ah yeah I had just copied the existing button from before. Makes sense to use route+model args. 

I wondered about just keeping it view secrets but I think it's nice UX because it creates a "move forward" notion after a secret has been synced. It felt a little awkward to have the "View secrets ->" link before the form has been submitted. 

Yankun approved this flow in her response to the gif I posted in our group chat.

Reply:
As you QA let me know if this feels awkward to you once this PR is merged and we can revisit!